salt/modules/inspectlib/collector.py

```diff
@@ -153,7 +156,7 @@ def _get_managed_files(self):
         links = set()
         files = set()
 
-        for line in os.popen("rpm -qlav").xreadlines():
+        for line in os.popen("rpm -qlav"):
```

Comment:
Oopsey, I just forgot to remove this, sorry. In fact, this part was made outside the Salt, because it was better fit to the environment, then I moved it later on. And, of course, forgot... Thanks!